app/assets/javascripts/discourse/app/components/sidebar/user/section-link.js

```diff
@@ -23,7 +24,23 @@ export default class SectionLink {
 
   @bind
   didStartDrag(e) {
-    this.mouseY = e.screenY;
+    if (e.button === 0) {
```

Comment:
What does 0 represent here?

Reply:
Should avoid these kind of magic numbers since the next person coming along has to figure out what it means too.